### PR TITLE
Only one word typo fix

### PR DESCRIPTION
**Pull request by KobayashiD27**
This commit fixes `Pre-pulling` to `Pre-pulled` in docs/concepts/containers/images.md .

### CODE REVIEW

File: content/en/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ template needs to include the `.docker/config.json` or mount a drive that contai
 All pods will have read access to images in any private registry once private
 registry keys are added to the `.docker/config.json`.
 
-### Pre-pulling Images
+### Pre-pulled Images
```
Comment (vanou) [inline content/en/docs/concepts/containers/images.md]: I think line 73 should be updated too.

File: content/en/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ in the Pod definition.
 All users of the cluster who can create pods will be able to run pods that use any of the
 images in the ECR registry.
 
-The kubelet will fetch and periodically refresh ECR credentials.  It needs the following permissions to do this:
+The kubelet eill fetch and periodically refresh ECR credentials.  It needs the following permissions to do this:
```
Comment (ryanmcginnis) [inline content/en/docs/concepts/containers/images.md]: change back to "will"

Comment (KobayashiD27) [reply content/en/docs/concepts/containers/images.md]: Thank you for your advice.

**Pull request merged**